addons/point_of_sale/models/pos_session.py

```diff
@@ -1642,7 +1642,6 @@ def _pos_ui_models_to_load(self):
             'res.users',
             'product.pricelist',
             'account.bank.statement',
-            'product.category',
```

Comment:
This is not okay. This is important in building the products.

Reply:
no need anymore in
```python
    def _process_pos_ui_product_product(self, products):
        """
        Modify the list of products to add the categories as well as adapt the lst_price
        :param products: a list of products
        """
        if self.config_id.currency_id != self.company_id.currency_id:
            for product in products:
                product['lst_price'] = self.company_id.currency_id._convert(product['lst_price'], self.config_id.currency_id,
                                                                            self.company_id, fields.Date.today())
        categories = self._get_pos_ui_product_category(self._loader_params_product_category())
        product_category_by_id = {category['id']: category for category in categories}
        for product in products:
            product['categ'] = product_category_by_id[product['categ_id'][0]]
```